hack/build-and-push.sh

```diff
@@ -172,6 +172,8 @@ do
       ANNOTATIONS+=("org.opencontainers.image.source=${VCS_URL}")
       ANNOTATIONS+=("org.opencontainers.image.revision=${VCS_REF}")
       ANNOTATIONS+=("org.opencontainers.image.url=${VCS_URL}/tree/${VCS_REF}/${task_dir}")
+      concrete_task_version=$(yq '.metadata.labels."app.kubernetes.io/version"' "$prepared_task_file")
+      ANNOTATIONS+=("org.opencontainers.image.version=${concrete_task_version}")
```

Comment:
Not all tasks have the version label I think. Will this do something sensible for those that don't (probably set the annotation to an empty string?)

Reply:
Updated. PTAL.